### ISSUE TITLE
Start address

### ISSUE THREAD

**Issue by jgauchia** [thalesfragoso/at28c-rs#2]
It's possible to add a command in CLI to indicate start address of EEPROM writing?

**Comment by thalesfragoso** [comment]
Yes, it should be doable. The firmware expects the address to read/write to be passed and the cli passes it starting from zero, but there is nothing in the design that prevents you from choosing the address if someone adds this feature.

**Comment by jgauchia** [comment]
OK, thank you
Maybe I'll try to add this feature.
